german/the-black-hack.md

```diff
@@ -101,7 +101,7 @@ The Black Hack verwendet keine Rettungswürfe und lässt die Spieler stattdessen
 
 |              INT               |            WEI            |           CHA           |
 |:------------------------------:|:-------------------------:|:-----------------------:|
-| Zaubern und Magie widerstehen. | Täuschung und Illusionen. | Geistige Beeinflussung. |
+| Zaubern und Magie widerstehen. | Täuschung und Illusionen. | Bezauberung. |
```

Comment:
Ich würde es so sehen:
* Täuschungen/Illusionen verändern die Wahrnehmung oder verfälschen irgendwie die Fakten, lassen dem Ziel aber alle Entscheidungs- und Handlungsfreiheit.
* "Charms" wirken direkt auf das Verhalten des Ziels, von emotionaler Beeinflussung bis hin zu echter "Gehirnwäsche", ohne dass etwas anderes verändert wird.
In Grenzfällen würde ich danach gehen, ob der freie Wille des Ziels irgendwie beeinflusst wird oder nicht.

"bezaubernd" würde ich zwar als gute Übersetzung für "charming" nehmen. Aber "Bezauberung" finde ich zu unspezifisch, um explizit damit zu benennen, was bei DSA unter "Beherrschungszauber" fällt. Aber tendenziell würde ich im D&D-Kontext an Jedi Mind Tricks denken, Pulp-Hypnose mit Pendeln und Drehspiralen, *Manchurian Candidate*, Besessenheit, solche Dinge...

Wörterbücher helfen da auch wenig, weil das englische "charm" heute inzwischen eher die Bedeutung von Schutzzauber/Segen/Amulett mit positiver Konnotation hat, weniger eine ungute Zauberwirkung durch magische Kräfte...